metrics/local.md

```diff
@@ -0,0 +1,29 @@
+# local
+
+The `local` metrics plugin stores the values on memory.
+
+## Parameters
+
+### `default_labels`
+
+Specifies the default labels for the metrics. \(default: `{agent: "Fluentd", hostname: "#{Socket.gethostname}"}`\)
+
+### `labels`
+
+Specifies other custom labels for the metrics. \(default: `{}`\)
+
+## Example
+
+With this configuration:
+
+```text
+<system>
+  <metrics>
+    @type local
+  </metrics>
+</system>
+```
+
+The above configuration will save the internal metrics for plugins on memory. As a result, you can retrive metrics from memory and also you can replace with your custom metrics plugin.
+
```

Comment:
So, It is important to explain `@type local` behavior here to clarify the difference.

Reply:
Yeah, we should do. 
Actually, `@type local` metrics plugin has equivalent to previous single value based Ruby instance variables.
This type of plugin is just used for storing numerical values on memory.

Reply:
Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/347/commits/db265fa15d5101cea7cb8b36d184eaa062d0afc9.